appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.0.{build}
+branches:
+  only:
+  - master
+image: Visual Studio 2017
+configuration: Release
+before_build:
+  appveyor-retry nuget restore
+build:
+  parallel: true
+  verbosity: minimal
+test:
+  assemblies:
+    - MetadataExtractor.Tests.dll
+test_script:
+  - dotnet test .\MetadataExtractor.Tests\MetadataExtractor.Tests.csproj --configuration Release --no-build
```

Comment:
I have added so test now also is compiling for net451 but that are not running due to path issue when finding file-assets for metadata extraction.

If we check one of the old builds that was working, example https://ci.appveyor.com/project/drewnoakes/metadata-extractor-dotnet/build/1.0.227 we can see that the test is running netcoreapp1.0-version. 

Make the net451 test running on ci-server can probably be added later and until that the test_script need to be added to run correct test-runner.